drivers/flash/soc_flash_mcux.c

```diff
@@ -119,7 +124,9 @@ struct flash_priv {
 };
 
 static const struct flash_parameters flash_mcux_parameters = {
-#if DT_NODE_HAS_PROP(SOC_NV_FLASH_NODE, write_block_size)
+#if CONFIG_FLASH_MCUX_LPC_IAP_WRITE_BUFFER
+	.write_block_size = 1,
```

Comment:
LPC55S69 does not have this property define. Do we need to add it?

Reply:
`write-block-size` is not defined for LPC55S69

Reply:
Within the DTS? It should be defined here: https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nxp/nxp_lpc55S6x_common.dtsi#L110